package.json

```diff
@@ -64,10 +68,12 @@
     "keysim": "2.0.1",
     "mkdirp": "0.5.1",
     "mocha": "5.1.1",
+    "mutation-observer": "^1.0.3",
```

Comment:
I see a number of these are being added but not used? Can you give me an idea of why these are required?

Reply:
This gets attached to the global test object as jsdom doesn't have a MutationObserver implementation by default. I've added a comment to this PR for where this happens.